src/NumericalAlgorithms/Interpolation/ObserveSurfaceIntegrals.hpp

```diff
@@ -0,0 +1,82 @@
+// Distributed under the MIT License.
```

Comment:
This could also be done later, as part of fixing issue #1095. If we do decide to postpone, let's add a note to that issue to remember this suggestion.

Reply:
postponing would be fine with me, but if this is quick to do in this PR then go ahead I'd say